docs/architecture/adr-102-rpc-companion.md

```diff
@@ -40,109 +40,128 @@ TBD
 ### Requirements
 
 The target audience for this solution are operators and integrators that want to alleviate the load on their nodes by offloading
-the queryable data requests to the **Standalone RPC**.
+the queryable data requests to the **RPC Companion**.
 
 This solution shall meet the following requirements in order to provide real benefits to these users.
 
-The **Standalone RPC** solution shall:
+The **RPC Companion** solution shall:
 
-1. Provide an **ingestion service** implemented as a data companion that can pull data from the node and store it on
+1. Provide an **[Ingest Service](#ingest-service)** implemented as a data companion that can pull data from the node and store it on
 its own storage (database)
-2. Provide its own storage (database) that can handle a high-load of reads and also have a good performance on inserting data.
-3. Implement a schema that needs to be backwards compatible with the current CometBFT RPC.
+2. Provide its own storage ([Database](#database)) that can handle a high-load of reads and also have a good performance on inserting data.
+3. Implement a [database schema](#database-schema) that needs to be backwards compatible with the current CometBFT RPC.
```

Comment:
actually I meant to say it would be compatible with Comet data structures

Reply:
Agree with Lasaro here. We need to ensure that the same data available through RPC can also be found on the database, using the same schema (data structures).

Reply:
In fact, it doesn't matter how we store data, provide that we can provide the same interface as the CometBFT RPC interface.

Reply:
yes, the Companion RPC interface (or a future gRPC) should provide wire compatible responses to the CometBFT own RPC interface. I think this can be re-worded better. Mixing the schema with RPC is indeed confusing. I'll try to make it more clear.